cairo_programs/benchmarks/secp_integration_benchmark.cairo

```diff
@@ -0,0 +1,6 @@
+from secp_integration_tests import run_tests
+
+func main{range_check_ptr}():
+    run_tests(0, 50)
```

Comment:
@Juan-M-V 
Did you check that running this program on the server takes between 100ms and 1second approx. ?
I think that it takes longer, but this is just an assumption since I don't run it